diff --git a/addons/counterstrikesharp/plugins/PlayerSkinMod/PlayerSkinMod.json b/addons/counterstrikesharp/plugins/PlayerSkinMod/PlayerSkinMod.json
index 482d590..f722e60 100644
--- a/addons/counterstrikesharp/plugins/PlayerSkinMod/PlayerSkinMod.json
+++ b/addons/counterstrikesharp/plugins/PlayerSkinMod/PlayerSkinMod.json
@@ -2,7 +2,7 @@
   "PluginProfile": {
     "Name": "PlayerSkinMod",
     "DisplayName": "Player Skin Mod",
-    "Version": "1.8.1",
+    "Version": "1.8.2",
     "Author": "CS2-Skin-local-mod",
     "Description": "Allow players to customize weapon skins, knives, gloves, agent models, music kits locally",
     "MinimumCounterStrikeSharpVersion": "1.0.0",
diff --git a/addons/counterstrikesharp/plugins/PlayerSkinMod/PlayerSkinModPlugin.cs b/addons/counterstrikesharp/plugins/PlayerSkinMod/PlayerSkinModPlugin.cs
index e6a7556..52b9c47 100644
--- a/addons/counterstrikesharp/plugins/PlayerSkinMod/PlayerSkinModPlugin.cs
+++ b/addons/counterstrikesharp/plugins/PlayerSkinMod/PlayerSkinModPlugin.cs
@@ -19,5 +19,5 @@ public class PlayerSkinModPlugin : BasePlugin
 {
     public override string ModuleName        => "PlayerSkinMod";
-    public override string ModuleVersion     => "1.8.1";
+    public override string ModuleVersion     => "1.8.2";
     public override string ModuleAuthor      => "CS2-Skin-local-mod";
     public override string ModuleDescription => "Allow players to customize weapon skins, knives, gloves, agent models, music kits locally";
